ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -1068,16 +1039,20 @@ Rectangle {
         Rectangle {
             id: messageInput
 
-            readonly property int defaultInputFieldHeight: 40
-
             Layout.fillWidth: true
-
             implicitHeight: inputLayout.implicitHeight + inputLayout.anchors.topMargin + inputLayout.anchors.bottomMargin
             implicitWidth: inputLayout.implicitWidth + inputLayout.anchors.leftMargin + inputLayout.anchors.rightMargin
 
             color: isEdit ? Theme.palette.statusChatInput.secondaryBackgroundColor : Style.current.inputBackground
             radius: 20
 
+            StatusQ.StatusToolTip {
+                id: messageLengthLimitTooltip
+                text: qsTr("Maximum message character count is " + control.messageLimit)
```

Comment:
That way the number can be put in the right place depending on the language
```suggestion
                text: qsTr("Maximum message character count is %1", control.messageLimit)
```

Reply:
And the correct form is actually like this :)

```qml
text: qsTr("Maximum message character count is %n", "", control.messageLimit)
```

Reply:
How about this one I just committed?

c4c7de7: Insert messageLimit no. via positional argument


https://github.com/status-im/status-desktop/pull/12082/files#diff-960651953687da8aeffbef44a532a34eaf8eeb0a5a1222773296640cebc0f81cR1051